turbopack/crates/turbopack-ecmascript/src/analyzer/mod.rs

```diff
@@ -396,8 +410,8 @@ pub enum JsValue {
     // ----------------------------
     /// A constant primitive value.
     Constant(ConstantValue),
-    /// An constant URL object.
-    Url(Box<Url>),
+    /// A constant URL object.
+    Url(ConstantString, JsValueUrlKind),
```

Comment:
I guess `ConstantString` should be `JsValue` here, so we could handle cases like `new URL(´./images/${expr}´, import.meta.url)`

Reply:
Makes sense, I'll do that in a follow up PR so that we can Worker support in first.